src/index.js

```diff
@@ -138,12 +145,19 @@ const init = function CozyBarInit ({
 } = {}) {
   i18n(lang)
   stack.init({cozyURL, token})
-  const view = injectDOM({lang, appName, appEditor, iconPath, replaceTitleOnMobile})
+  view = injectDOM({lang, appName, appEditor, iconPath, replaceTitleOnMobile})
 
   if (view) {
     bindEvents.call(view)
     view.on('teardown', unbindEvents.bind(view))
   }
 }
 
-module.exports = { init, version: __VERSION__ }
+// set the cozy bar locale from the application
+const setLocale = function SetLocale (lang) {
```

Comment:
I really don't know for now but all functions and methods are declared in this way in the cozy bar code so I just followed to be coherent.